crates/matrix-sdk/src/oidc.rs

```diff
@@ -781,6 +851,23 @@ pub(crate) struct OidcData {
     pub(crate) latest_id_token: RwLock<Option<IdToken<'static>>>,
 }
 
+/// Information about the URL that should be presented to authenticate with the
+/// server.
+#[derive(Debug)]
+pub struct OidcAuthenticationUrl {
```

Comment:
I'm not 100% sure if this is the correct name for this struct, totally open to suggestions.

Reply:
Maybe `OidcAuthorizationData`?

OIDC in general uses the term "authorization", not "authentication"

Reply:
Good shout 👍